src/plugins/oSnap/utils/getters.ts

```diff
@@ -47,6 +47,8 @@ async function callGnosisSafeTransactionApi<TResult = any>(
   network: Network,
   url: string
 ) {
+  if(!GNOSIS_SAFE_TRANSACTION_API_URLS[network])
+    throw new Error(`No gnosis safe api defined for network ${network}`)
```

Comment:
throw a more obvious error when we are missing an api url in the constant object